demos/notebooks/demo_pipeline.ipynb

```diff
@@ -4,13 +4,34 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<html><head><meta content=\"text/html; charset=UTF-8\" http-equiv=\"content-type\"><style type=\"text/css\">ol</style></head><body class=\"c5\"><p class=\"c0 c4\"><span class=\"c3\"></span></p><p class=\"c2 title\" id=\"h.rrbabt268i6e\"><h1>CaImAn&rsquo;s Demo pipeline</h1></p><p class=\"c0\"><span class=\"c3\">This notebook will help to demonstrate the process of CaImAn and how it uses different functions to denoise, deconvolve and demix neurons from a two-photon Calcium Imaging dataset. The demo shows how to construct the `params`, `MotionCorrect` and `cnmf` objects and call the relevant functions. You can also run a large part of the pipeline with a single method (`cnmf.fit_file`). See inside for details.\n",
+    "# CNMF demo pipeline: Intro\n",
+    "This demo presents a full pipeline for the analysis of a two-photon calcium imaging dataset using the CaImAn (**Ca**lcium **Im**aging **An**alysis) software package. It demonstrates how to use Caiman's built-in tools for the following three analysis steps:\n",
     "\n",
-    "Dataset couresy of Sue Ann Koay and David Tank (Princeton University)\n",
```

Comment:
We might want to keep this "thank you" notice

Reply:
I kept this but moved it to 'specify data to be processed' section that includes a fuller discussion of the data.